src/diff/index.js

```diff
@@ -1,11 +1,40 @@
 import { EMPTY_OBJ, EMPTY_ARR } from '../constants';
-import { Component } from '../component';
+import { Component, getDomSibling } from '../component';
 import { Fragment } from '../create-element';
 import { diffChildren } from './children';
 import { diffProps, setProperty } from './props';
 import { assign, removeNode } from '../util';
 import options from '../options';
 
+const reorderChildren = newVNode => {
+	let lastVNodeChild;
+	for (let tmp = 0; tmp < newVNode._children.length; tmp++) {
+		if (newVNode._children[tmp]) {
+			newVNode._children[tmp]._parent = newVNode;
+			lastVNodeChild = newVNode._children[tmp];
+		}
+	}
+
+	if (
+		lastVNodeChild &&
+		lastVNodeChild.type &&
+		lastVNodeChild._dom &&
+		lastVNodeChild._dom.isConnected
```

Comment:
What's the reason for checking `isConnected` here? I worry because I think folks sometimes call render() with a parent element that is not yet connected, so the whole tree would have `isConnected=false`.

Reply:
I added this check in one of my explorations to fix some failing suspense tests though I don't understand why they were failing and why this fixed them. So all that to say, yes, this line deserves further investigation